tests/contrib/test_zoho.py

```diff
@@ -42,18 +42,6 @@ def test_load_from_config(make_app):
     assert client_id == "foo"
 
 
-@responses.activate
-def test_load_from_params(make_app):
-    app = make_app(client_id="not_foo", client_secret="not_bar")
-    app.config["ZOHO_OAUTH_CLIENT_ID"] = "foo"
-    app.config["ZOHO_OAUTH_CLIENT_SECRET"] = "bar"
-
-    resp = app.test_client().get("/zoho")
-    url = resp.headers["Location"]
-    client_id = URLObject(url).query.dict.get("client_id")
-    assert client_id == "not_foo"
-
```

Comment:
Did you intend to remove this test?

Reply:
Yes, I did. It was asserting that the params passed in the `create_zoho_blueprint()` function should override the values in the app config. This does not match the behavior with any of the other blueprints, so I changed the behavior and deleted the test.

Do you think it was the right thing to do? I'm curious to hear your opinion.

Reply:
Makes sense. I always value consistency over magical unique feature.